clusterloader2/testing/prometheus/scrape-etcd.yaml

```diff
@@ -0,0 +1 @@
+PROMETHEUS_SCRAPE_ETCD: true
```

Comment:
Does this enable it? If so, we shouldn't do that, right?

Reply:
It doesn't, but we will use it to enable etcd scraping in the test-infra repo by having:
```
      - --test-cmd-args=--testoverrides=./testing/prometheus/scrape-etcd.yaml
```